automated install/basic-install.sh

```diff
@@ -165,11 +167,11 @@ if command -v apt-get &> /dev/null; then
     # Check for and determine version number (major and minor) of current php install
     if command -v php &> /dev/null; then
         phpInsVersion="$(php -v | head -n1 | grep -Po '(?<=PHP )[^ ]+')"
-        echo -e "  ${INFO} Existing PHP installation detected : PHP version $phpInsVersion"
-        phpInsMajor="$(echo "$phpInsVersion" | cut -d\. -f1)"
-        phpInsMinor="$(echo "$phpInsVersion" | cut -d\. -f2)"
+        printf "  %b Existing PHP installation detected : PHP version %s\\n" "${INFO}" "${phpInsVersion}"
```

Comment:
It can, but there are so many other places that use the double quote that I'd prefer not to have singleton's out there with single quotes. A lot of the `printf` calls will be functionalized anyways to have something like a `warn` `error` and `fatal` function that prints to the terminal and then dumps sanitized text to a log file.

Reply:
Marking this as resolved.